packages/sign_in_with_apple/darwin/Classes/SignInWithAppleError.swift

```diff
@@ -90,6 +107,8 @@ public enum SignInWithAppleError {
                 errorCode = "authorization-error/notHandled"
             case .failed:
                 errorCode = "authorization-error/failed"
+            case .notInteractive:
+                errorCode = "authorization-error/notInteractive"
```

Comment:
This error code should be mapped on the Dart side in https://github.com/aboutyou/dart_packages/blob/71c50aa29931c925c95edbec8ee9b2ca9df42ca2/packages/sign_in_with_apple/lib/src/exceptions.dart#L42